app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -119,9 +121,21 @@ def instance_storage(persister_hardware, instance)
   def instance_operating_system(persister_instance, instance)
     image_id = instance&.dig(:image, :id)
     os = img_to_os[image_id] || pub_img_os(image_id)
+    os_name = normalize_os(os)
     persister.operating_systems.build(
       :vm_or_template => persister_instance,
-      :product_name   => os
+      :product_name   => os_name,
+      :version        => os
+    )
+  end
+
+  def image_operating_system(persister_image, image)
+    os_name = image&.dig(:operating_system, :name)
+    normalized_os_name = normalize_os(image&.dig(:operating_system, :name))
+    persister.operating_systems.build(
+      :vm_or_template => persister_image,
+      :product_name   => normalized_os_name,
+      :version        => os_name
```

Comment:
Future refactoring, looks like we could combine these methods a bit by doing something like:
```
def instance_operating_system(persister_instance, instance)
  image_id = instance.dig(:image, :id)
  image = collector.images_by_id[image_id] || collector.image(image_id)
  vm_or_template_operating_system(persister_instance, image)
end
```

Since the only difference is looking up the image from the instance

Reply:
@chadh1313 once you update for https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/113/files#r516935249 I'll take up this refactoring in a follow-up PR

Reply:
Sounds good - the spec cleanup and update for:  https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/113/files#r516935249 are pushed.